packages/discord-request/src/types.ts

```diff
@@ -56,3 +56,15 @@ export interface RateLimitData {
 	global: boolean;
 	method: RequestMethod;
 }
+
+// TODO: implement getMany and setMany.
+// I feel like the T typing here is a bit messy as well,
+// since redis only seems to return strings :/
```

Comment:
Does the lib do this? My understanding was the Redis type was STRING, but some libraries would automatically convert it to Number

Reply:
Nope unfortunately not, so if we want to get numbers back we have to convert it to numbers ourselves